### ISSUE TITLE
[Bug]:  This mod failed in the Player.InventoryChanged event

### ISSUE THREAD

**Issue by argon932** [Annosz/UIInfoSuite2#413]
### Game Version

v2.3.0

### Multiplayer

Single Player

### SMAPI Log

https://smapi.io/log/e01de7baf3fb4435ac5b52bdfb79eb9b

### Steps to Reproduce
1. Hover over an item you can donate to community center, repeat
2. Check SMAPI log

**Comment by Zamiell** [comment]
This is likely a bug/conflict with some other mod, as the long-time feature of the "community center tooltip" seems to be working just fine for me on the latest version.

**Comment by argon932** [comment]
Yeah, I was just testing it and now can't repeat it. Strange! If I find the cause I will re-open.

**Comment by drewhoener** [comment]
@argon932 I also notice from your log that you used the debug bundle command, I wonder if somehow that messed with it checking the loaded bundles.
Regardless it probably shouldn't throw like that, I'll see if I can add a guard so it doesn't spam errors
